bots/running-a-rasa-bot/README.md

```diff
@@ -1,54 +1,33 @@
 # Running a Rasa Bot
 
 Rasa is the leading open-source machine learning toolkit that lets developers expand bots beyond answering simple questions
-with minimal training data. The bots are based on a machine learning model trained on example conversations.
+with minimal training data. At the core Rasa bot has a machine learning model which trained on example conversations.
 
 ## Quickstart
 
-Rasa is developed with Python. For the most part you dont need to know python to design the basic conversational flows in Rasa.
+Rasa is developed with Python, but for the most part you dont need to know python to design the basic conversational flows in Rasa.
 You will need python knowledge when creating custom actions in Rasa which calls external actions. The fastest way to start with
 Rasa is by using [Rasa starter kit](https://github.com/RocketChat/rasa-kick-starter). The starter kit uses webhooks to
 communicate between Rasa and Rocket.Chat.
 
-### 1. Install Rocket.Chat
+Clone the quick start guide with the following command to get started  with the setup.
 
-If you already have a Rocket.Chat instance you can skip this step
+`git clone https://github.com/RocketChat/rasa-kick-starter`
 
-```sh
-docker-compose up -d mongo
-docker-compose up -d mongo-init-replica
-docker-compose up -d rocketchat
-```
-
-Access <http://localhost:3000/>
-
-Create an `admin` user with the password `admin`.
-
-### 2. Rocket.Chat Bot User Configurations
-
-Clone the quick start guide with the following command: `git clone https://github.com/RocketChat/rasa-kick-starter`
-Create a Rasa bot user in Rocket.Chat. You can either manually login to Rocket.Chat and create a bot user via the
-user management page or can use the following script to create the bot user.
+### Create Rocket.Chat Bot
```

Comment:
All the steps related to creating a bot user are already described here: https://rocket.chat/docs/bots/create-and-run-a-bot/#steps

You can safely delete this section